configure.ac

```diff
@@ -38,6 +38,13 @@ AC_ARG_ENABLE(
 	[enable_distonly="no"]
 )
 
+AC_ARG_ENABLE(
+	[password-save],
+	[AS_HELP_STRING([--disable-password-save], [disable password save support @<:@default=yes@:>@])],
+	[enable_password_save="yes"],
+	[enable_password_save="no"]
+)
```

Comment:
On Mon, Aug 1, 2016 at 1:20 PM, Ilya Shipitsin <notifications@github.com>
wrote:

> > @@ -38,6 +38,13 @@ AC_ARG_ENABLE(
> >  	[enable_distonly="no"]
> >  )
> >
> > +AC_ARG_ENABLE(
> > +	[password-save],
> > +	[AS_HELP_STRING([--disable-password-save], [disable password save support @<:@default=yes@:>@])],
> > +	[enable_password_save="yes"],
> > +	[enable_password_save="no"]
> > +)
>
> I tried it with "yes" in the last arg and it does not work as expected.
> on the other side,
> https://travis-ci.org/OpenVPN/openvpn-gui/builds/148929302 works exactly
> as expected
>
> I agree that we can omit default value. however, it is not clear about
> yes/no choice
>
No, the third arg is not the default, the 4th is the default. Set the 4th
to our default (yes) and third as empty, and it should work.

In AC_ARG_ENABLE the last arg defines the default value to be used if
neither --enable-xxx or --disable-xxx is passed to ./configure. So that
should be "yes" in our case as --enable-password-change is the default.

The third arg is used when the option is specified in either--enable or
--disable form -- in this case enable_password_change will already have the
correct value so that arg should be empty unless some other checks are
required.

Selva